javascript/core/src/verbs/stepFunctions/simpleSteps.ts

```diff
@@ -0,0 +1,133 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { all, escape, op } from 'arquero'
+import type ColumnTable from 'arquero/dist/types/table/column-table'
+
+import type { TableStep } from '../nodeFactories/index.js'
+import type {
+	AggregateArgs,
+	BinarizeArgs,
+	BooleanArgs,
+	DedupeArgs,
+	EraseArgs,
+	FillArgs,
+	FilterArgs,
+	FoldArgs,
+	GroupbyArgs,
+	ImputeArgs,
+	PivotArgs,
+	RecodeArgs,
+	RenameArgs,
+	RollupArgs,
+	SampleArgs,
+	SelectArgs,
+	SpreadArgs,
+	UnrollArgs,
+	WindowArgs,
+} from '../types/domainTypes.js'
+import { deriveBoolean } from '../util/expressions.js'
+import { compareAll, singleExpression } from '../util/index.js'
+import type { ExprFunctionMap } from '../util/types.js'
+
+export const aggregateStep: TableStep<AggregateArgs> = (
```

Comment:
I think I prefer having the steps broken into individual files. At the expense of more file management it is easier to fine and open files, especially when they are grouped into "types" like "simple steps" that may not be immediately apparent

Reply:
Yeah I went back and forth on this. A bunch of tiny files was kind of annoying but I get the argument against clumping them. I like separating the stepFunctions from the node factories (it makes it easier to test), but this means that you don't have a full, canonical "verb list".

Would breaking up simpleSteps.ts into separate files satisfy your concern here?

Reply:
I think so - just a flat list of verb files seems the easiest to follow

Reply:
sure thing, I'm fine with one-file per verb